ui-backend/catalog-plugin-highlight/src/main/java/org/codice/ddf/catalog/ui/plugin/highlight/HighlightTransformPlugin.java

```diff
@@ -116,9 +118,12 @@ private void processHighlight(
       String attributeName,
       Highlight highlight) {
     Attribute attribute = matchingResult.getMetacard().getAttribute(attributeName);
-    String value = (String) attribute.getValues().get(highlight.getValueIndex());
+    String value = null;
+    if (attribute != null && !attribute.getValues().isEmpty()) {
+      value = (String) attribute.getValues().get(highlight.getValueIndex());
+    }
 
-    if (!value.equals("REDACTED")) {
+    if (value != null && !value.equals("REDACTED")) {
```

Comment:
If we add the per Source Point of Contact information then this will contain more than "REDACTED".
Would using `value.contains()` make more sense?
Also would be safer if we are referring to a common constant for the string "REDACTED" - not sure where this would live?